packages/open-lens/src/common/sample-bundled-extension.injectable.ts

```diff
@@ -0,0 +1,15 @@
+import { getInjectable } from "@ogre-tools/injectable";
+import { bundledExtensionInjectionToken } from "@k8slens/core/common";
+import manifest from "@k8slens/sample-extension/package.json";
+
+const cliLensExtensionInjectable = getInjectable({
```

Comment:
```suggestion
const sampleBundledExtensionInjectable = getInjectable({
```
similar for id and export